src/audio/buffer.c

```diff
@@ -75,8 +75,8 @@ struct comp_buffer *buffer_new(struct sof_ipc_buffer *desc)
 {
 	struct comp_buffer *buffer;
 
-	tr_info(&buffer_tr, "buffer new size 0x%x id %d.%d", desc->size,
-		    desc->comp.pipeline_id, desc->comp.id);
+	tr_info(&buffer_tr, "buffer new size 0x%x id %d.%d flags %d",
```

Comment:
%x would give more readable output in case of bitfield.

Reply:
Good tip. Done.